src/inputs/DateTimeInput.js

```diff
@@ -106,6 +106,7 @@ class DateTimeInput extends BaseInput {
       initializeWith: getInitializer({ initialDate, dateFormat: dateTimeFormat, dateParams: this.getDateParams() }),
       value: parseValue(chooseValue(value, initialDate), dateTimeFormat),
       disable: parseArrayOrValue(disable),
+      preserveViewMode: true,
```

Comment:
This is unnecessary to provide ``preserveViewMode`` to picker (picker knows nothing about ``mode``).

Reply:
right, this would be the prop on the component that would allow the developer to specify whether the `onFocus` action is the initial view mode state (`preserveViewMode: false`) or the last view mode (`preserveViewMode: true`)